src/core/Context.js

```diff
@@ -37,12 +37,13 @@ define(function(require, exports, module) {
      * @private
      * @param {Node} container Element in which content will be inserted
      */
-    function Context(container) {
+    function Context(container, appMode) {
```

Comment:
I am unsettled with passing the appMode flag as a parameter, but I didn't want to create a circular dependency between Engine and Context so...